content/en/blog/2021/etcd-container-linux-with-systemd.md

```diff
@@ -1,10 +1,11 @@
 ---
 title: Container Linux with systemd
-weight: 7200
+author: Gyuho Lee, Somoshree Datta
```

Comment:
Just my thought but I would only keep Gyuho's name here, considering the technical content, if we really need author.  Also, same comment with date as my previous comment in a related PR. Thanks @Somoshree @nate-double-u

Reply:
In https://github.com/etcd-io/website/pull/468 we've added a note at the bottom of the page explaining the provenance of the page. I think we can do that here too, remove the author line, and add a similar note here.